### DISCUSSION TITLE
✅ 🔗 Add custom code feature to frontend sidebar editor (Duplicate)

### DISCUSSION THREAD

**Discussion by abobakrd** [elementor/elementor#14480] [Feature Request]
Just like the CSS editor that's available at template/post level in the sidebar editor in frontend, a script editor for inline JS would also be really useful.

Why?
- HTML widgets with scripts gets lost in big templates/custom posts layout
- HTML widgets with code that removes or hides HTML elements within the widget takes away the widget visibility in the frontend editor
- The new Custom Code feature seems more deemed for use of multiple templates/posts considering the conditional ruleset feature, making it tedious for single-instance-use.

Just an idea! It's obviously not very necessary, but it would certainly make adding inline JS at post/template level much easier, just like the useful post/template CSS editor. 👍

**Comment by nicholaszein** [comment]
Hello @abobakrd!

This feature was introduced with Elementor Pro version 3.1, back in February, 2021. It's called Custom Code.

https://elementor.com/help/custom-code-pro/

I'm closing the request as solved.

Kind regards
